kured-ds-windows.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kured
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:

```

Comment:
How would the node locking work with two damonsets?  As implemented, it would allow a windows node and linux node to reboot at the same time.  Perhaps this isn't necessarily an issue.